server/graphql/v1/mutations/collectives.js

```diff
@@ -590,10 +590,7 @@ export async function claimCollective(_, args, req) {
   // create default tiers
   const host = await models.Collective.findByPk(defaultHostCollective('opensource').CollectiveId);
 
-  collective = await collective.addHost(host, {
-    ...req.remoteUser.minimal,
-    isAdmin: () => true,
```

Comment:
I also removed this pattern to instead introduce a `shouldAutomaticallyApprove` boolean in `options`